CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## 0.5.0
+
+- Setting `preserve_permissions` to `false` will avoid setting _any_ permissions on extracted files.
+  In [`alexcrichton/tar-rs`](https://github.com/alexcrichton/tar-rs), setting `preserve_permissions`
+  to `false` will still set read, write, and execute permissions on extracted files, but will avoid
+  setting extended permissions (e.g., `setuid`, `setgid`, and `sticky` bits).
+- Ban unpacking files, directories or references that are or point to locations outside the unpack
```

Comment:
I think we already did this. The change we ported ensured that we didn't create _directories_ outside of the target location. (We already avoided files, it's just that we were creating directories _prior_ to validating the file target.)